xrdp/xrdp_mm.c

```diff
@@ -1257,11 +1369,13 @@ xrdp_mm_drdynvc_up(struct xrdp_mm *self)
             !g_text2bool(enable_dynamic_resize))
     {
         LOG(LOG_LEVEL_INFO, "User has disabled dynamic resizing.");
+        return error;
     }
-    else
-    {
-        error = dynamic_monitor_initialize(self);
-    }
+    ignore_marker = (struct display_size_description *)
+                    g_malloc(sizeof(struct display_size_description), 1);
```

Comment:
The above two should be `struct dynamic_monitor_description` as mentioned above.

Reply:
Fixed.